### PR TITLE
sig-cluster: introduce some responsibilities

### PR DESCRIPTION
**Pull request by enp0s3**
Signed-off-by: Igor Bezukh <ibezukh@redhat.com>

### What this PR does
Continue the separation of ownership concerns effort, hereby introduce sig-cluster.

### Checklist

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

### Release note

```release-note

```

### CODE REVIEW

**Comment by fabiand** [comment]
Thanks @enp0s3 please also add labels in each OWNERS file ot set labels accoridng to SIG.

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -152,3 +152,10 @@ aliases:
   #
   sig-architecture-approvers: []
   sig-architecture-reviewers: []
+  
+#
+# SIG Cluster
+# Owns the cluster level application code related to Kubevirt i.e.
+# API, CRD controllers, cert-rotation, operator, etc.
+sig-cluster-approvers: []
+sig-cluster-reviewers: []
```
Comment (jean-edouard) [inline OWNERS_ALIASES:161]: Missing indent?

Comment (fabiand) [reply OWNERS_ALIASES:161]: After discussion on ml and with Lubo and Antonio, we agreed to stick to sig-compute for now, and remerge sig-node and sig-cluster usage in OWNER files into sig-compute again.

The split will be picked up async again.

**Review by fabiand — COMMENTED** [review]
Please rework thi PR to
1. include labels (sig/compute)
2. And as discussed, merge sig-node and sig-cluster into sig-compute (again!)

**Pull request closed without merging**